jet-fuel/config/routes.rb

```diff
@@ -0,0 +1,4 @@
+Rails.application.routes.draw do
+  root 'welcome#index'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.